lib/openstudio-standards/standards/Standards.Model.rb

```diff
@@ -445,7 +445,7 @@ def model_create_prm_any_baseline_building(user_model, building_type, climate_zo
         next if plant_loop_swh_loop?(plant_loop)
 
         plant_loop_apply_prm_number_of_boilers(plant_loop)
-        plant_loop_apply_prm_number_of_chillers(plant_loop, sizing_run_dir)
+        plant_loop_apply_prm_number_of_chillers(model, plant_loop)
```

Comment:
The CI tests are failing because the `plant_loop_apply_prm_number_of_chillers` method is expecting (OpenStudio::Model::PlantLoop, String) has input arguments, not (OpenStudio::Model::Model, OpenStudio::Model::PlantLoop)

Reply:
Yup! Fixing that now, thanks Matt.